### ISSUE TITLE
Make output more machinereadable

### ISSUE THREAD

**Issue by rliebig** [hhucn/webvulnscan#41]
The current output would  be difficult to parse. The format should look like the following:

```
Vulnerability: <Name of the Vulnerability> <site> <additional message>
```

Also, the log messages should be adjusted. The type of the vulnerability should directly be something like "XYZ Injection" and not "XYZ Vulnerability".

**Comment by rliebig** [comment]
I just though about this - this would be nonsense. It would be more logically:

```
Vulnerability: <site> <Name of ther Vulnerabilit> <additional message>
```

...

**Comment by phihag** [comment]
How's that any different from the current output?

**Comment by rliebig** [comment]
The notation I used is a bit unclear. It should look like:

```
Vulnerability: http://test <Name of the Vulnerabilit> additional message
```

**Comment by rliebig** [comment]
Closed via 675fd78
